### PR TITLE
Added fast-query-parsers as a package

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by thewchan** [comment]
@conda-forge/staged-recipes might need some Rust help on why the win build is failing

**Comment by thewchan** [comment]
@conda-forge/staged-recipes this is ready for review; win build fails due to dependencies not available in that platform but this is noarch anyway.

**Comment by ocefpaf** [comment]
> win build fails due to dependencies not available in that platform but this is noarch anyway.

For compiled extensions, different from noarch, you have to skip Windows to avoid triggering the Windows CI all the time.

File: recipes/fast-query-parsers/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fast_query_parsers-{{ version }}.tar.gz
+  sha256: df972c0b58d0bf51fa43b67d2604ab795984015d47552d02175ebcc685e4852b
+
+build:
+  skip: True  # [py<38]
```
Comment (ocefpaf) [inline recipes/fast-query-parsers/meta.yaml:13]:
```suggestion
  skip: True  # [py<38] or [win]
```

Comment (thewchan) [reply recipes/fast-query-parsers/meta.yaml]: My bad I was doing another  recipe that was noarch and got confused

**Comment by thewchan** [comment]
@ocefpaf seems like at the mean time someone else created the feedstock. Thanks anyway!

**Pull request closed without merging**